sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Infrastructure/TestEnvironment.cs

```diff
@@ -115,5 +115,15 @@ private static string CreateUri(string fileName, string assetFolder = default, s
         {
             return string.Format(fileUriFormat ?? FileUriFormat, assetFolder ?? AssetsFolderName, fileName);
         }
+
+        /// <summary>
+        /// Retrieves the URI string to a specified file.
+        /// </summary>
+        /// <param name="filename">The name of the file to retrieve.</param>
+        /// <returns>The URI string to the specified file.</returns>
+        public static string RetrieveUri(string filename)
```

Comment:
In a previous PR I created a `CreateUri` method which is basically the same. Do you mind using that one instead?

Reply:
Not at all - thanks for the pointer!

Reply:
I'll make it public so it's accessible from samples.